src/Database/DBStructure.php

```diff
@@ -207,7 +207,7 @@ private static function printUpdateError($message) {
 	public static function update($verbose, $action, $install = false, array $tables = null, array $definition = null) {
 		if ($action && !$install) {
 			Config::set('system', 'maintenance', 1);
-			Config::set('system', 'maintenance_reason', L10n::t(': Database update', DBM::date().' '.date('e')));
+			Config::set('system', 'maintenance_reason', L10n::t('%1: Database update', DBM::date().' '.date('e')));
```

Comment:
Didn't you mean `%s`?

Reply:
Oops ... Yeah. The placeholders in the language at our company are looking that way.